src/interfaces/Config/ICloudinaryConfigurations.ts

```diff
@@ -0,0 +1,12 @@
+import IAccountConfig from "./IAccountConfig";
+import ICloudinaryAssetConfigurations from "./ICloudinaryAssetConfigurations";
+import ITagConfig from "./ITagConfig";
+import IApiConfig from "./IApiConfig";
+
+interface ICloudinaryConfigurations extends ICloudinaryAssetConfigurations{
+  account?: IAccountConfig
+  tag?: ITagConfig
+  api?: IApiConfig
```

Comment:
Where are `URLConfig` and `CloudConfig`?

Reply:
Inside ICloudinaryAssetConfigurations

(notice this is an extension)